person/views.py

```diff
@@ -118,7 +121,32 @@ def asem_user_details(request, asem_user_id):
     asem_user.status = choices_dict['status'][asem_user.status]
     asem_user.own_home = choices_dict['housing_type'][asem_user.own_home]
 
-    return render(request, 'asem_user/asem_user_details.html', {'asem_user': asem_user})
+    fields = [f for f in ASEMUser._meta.get_fields() if f.name not in ["id", "photo", "password", "user_type", "name", "surname", "service", "ong", "person_ptr"]]
+    info = [getattr(asem_user, f.name) for f in fields]
+
+    fields_info = dict(zip([f.verbose_name for f in fields], info))
+    
+    items = list(fields_info.items())
+
+    for item in items:
+        if(item[1] == True):
```

Comment:
As a suggestion, some comments could be added here to know why are the different if statements

Reply:
What is the problem here you don't understand? If you could be more specific, I would be glad to add more comments 😃

Reply:
😳I thought it would be useful to point why are every line but it is ok like this